omniledger/service/api.go

```diff
@@ -168,6 +172,31 @@ func (c *Client) GetChainConfig() (*ChainConfig, error) {
 	return config, nil
 }
 
+// WaitProof will poll omniledger until a given instanceID exists.
```

Comment:
s/omniledger/OmniLedger/

What's up with you guys, do I need to make a pre-commit hook for this? :)

Reply:
REAL question: or is it `Omniledger`, like the java-class?

Reply:
In order to sync up to the paper, I made a sweep thru a while ago making all the Go code OmniLedger.

Reply:
should we do the same for java?

Reply:
made `OmniLedger` in all .md and .go in omniledger-directory.